db/models/articles-model.js

```diff
@@ -97,6 +97,47 @@ exports.createArticleComment = (article_id, username, body) => {
         }  
     })
 }
-// exports.updateArticleById =() => {
-
-// }
\ No newline at end of file
+exports.updateArticleById =(votes_inc, article_id) => {  
+    if(isNaN(votes_inc)) {
+        return Promise.reject({
+            status: 400, message: 'invalid data type'
+        })
+    }
+    if(isNaN(article_id)) {
+        return Promise.reject({
+            status: 400, message: 'article_id must be a number'
+        })
+    }
+    return db
+    .query(
```

Comment:
Similarly here, we shouldn't need a separate database query to check if the article exists. When running the patch query we will likely be violating a foreign key restraint if the id doesn't exist on the articles table.

Reply:
done